tests/templates/kuttl/snapshot-export/31-run-tests.yaml.j2

```
@@ -0,0 +1,41 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: test-snapshot-export
+spec:
+  template:
+    spec:
+      containers:
+        - name: hbase
+          image: docker.stackable.tech/sandbox/hbase:{{ test_scenario['values']['hbase'] }}-stackable0.0.0-pr621
```

Change when stackabletech/docker-images#621 is merged:

```suggestion
          image: docker.stackable.tech/sandbox/hbase:{{ test_scenario['values']['hbase'] }}-stackable0.0.0-dev
```

---

Fixed in 7dd1592eb993453b1ed6f3cf8fed5ba77c0ba531